docs/writing-docs/doc-block-argstable.md

```diff
@@ -30,9 +30,9 @@ This is extremely useful, but it can be further expanded. Additional information
 
 The args tables will be updated accordingly by including the additional information (e.g., JSDocs comments), offering a richer experience for any stakeholders involved.
 
-## Working with the DocsPage
+## Working with Docs
 
-To use the `ArgsTable` in [DocsPage](./docs-page.md#component-parameter), export a component property on your stories metadata:
+To use the `ArgsTable` in Storybook's Docs, export a component property on your stories metadata:
 
 <!-- prettier-ignore-start -->
 
```

Comment:
@JReinhold — I cannot comment directly on it, but for the props table below, is `of` correct? I figured it would match `<Meta>` and accept the CSF meta. And, if that's correct, is `story` deprecated (or will be soon)?

```md
| Option  | Description                                                                                         |
| ------- | --------------------------------------------------------------------------------------------------- |
| `of`    | Infers the args table from the component. <br/> `<ArgsTable of={MyComponent} />`                    |
| `story` | Infers the args table based on a story. <br/> `<ArgsTable story="example-mycomponent--my-story" />` |
```

Reply:
Would appreciate you weighing in on any other Docs Blocks API changes, while you're here. 🙂